### PR TITLE
Information per graph

### PR DESCRIPTION
**Pull request by alegnaaived**
Defining classes and components to show graphs information. Applied just to BarStackHorizontal

### CODE REVIEW

File: src/commons/ShortInfo.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    return (
+      <div>
+        <div
+          className={`${className}-${hideText}`}
+        >
+          <p>
+            <b>
+              {name}
+            </b>
+            {description && (` ${description}`)}
```
Comment (erikasv) [inline src/commons/ShortInfo.jsx]: The validation isn't necessary, just with {\` ${description}\`} is enough

File: src/commons/ShortInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  name: PropTypes.string.isRequired,
+  className: PropTypes.string,
+  description: PropTypes.string,
+  tooltip: PropTypes.string.isRequired,
+  customButton: PropTypes.bool,
+};
+
+ShortInfo.defaultProps = {
+  className: 'hidden',
+  description: NaN,
```
Comment (erikasv) [inline src/commons/ShortInfo.jsx]: Since it's an string its default value should be null or an empty string

File: src/commons/ShortInfo.jsx (hunk 1 above)
Comment (alegnaaived) [reply src/commons/ShortInfo.jsx]: Fixed

File: src/commons/ShortInfo.jsx (hunk 2 above)
Comment (alegnaaived) [reply src/commons/ShortInfo.jsx]: Fixed

**Pull request merged**